vscode-web/main.tf

```
@@ -59,6 +59,12 @@ variable "install_prefix" {
   default     = "/tmp/vscode-web"
 }
 
+variable "vscode_web_commit_id" {
```

```suggestion
variable "commit_id" {
```
We can shorten the name.